src/style.css

```diff
@@ -0,0 +1,272 @@
+}
+
+.day7::before {
+  content: '7';
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+.day8::before {
+  content: '8';
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+.day9::before {
+  content: '9';
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+.day10::before {
+  content: '10';
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+.day11::before {
+  content: '11';
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+.day12::before {
+  content: '12';
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+.day13::before {
+  content: '13';
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+.day14::before {
+  content: '14';
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+.day15::before {
+  content: '15';
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+.day16::before {
+  content: '16';
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+.day17::before {
+  content: '17';
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+.day18::before {
+  content: '18';
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+.day19::before {
+  content: '19';
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+.day20::before {
+  content: '20';
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+.day21::before {
+  content: '21';
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+.day22::before {
+  content: '22';
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+.day23::before {
+  content: '23';
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+.day24::before {
+  content: '24';
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+.day25::before {
+  content: '25';
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+.day26::before {
+  content: '26';
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+.day27::before {
+  content: '27';
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+.day28::before {
+  content: '28';
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+.day29::before {
+  content: '29';
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+.day30::before {
+  content: '30';
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+.day31::before {
+  content: '31';
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+.day--mon {
+  margin-left: 0;
+}
+
+.day--tue {
+  margin-left: 105px;
+}
+
+.day--wed {
+  margin-left: 205px;
+}
+
+.day--thu {
+  margin-left: 305px;
+}
+
+.day--fri {
+  margin-left: 405px;
+}
+
+.day--sat {
+  margin-left: 505px;
+}
+
+.day--sun {
+  margin-left: 605px;
+}
+
+.calendar .month--length:nth-child(28) {
```

Comment:
The selector `.calendar .month--length:nth-child(28)` is intended to set opacity for certain days, but the HTML file uses `day__month--length` as a class. Ensure that the class names match between the HTML and CSS files.